File: PHP/Contoh Fungsi/Fungsi (Functions) Hitung/Fungsi (Functions) Hitung.php
<!DOCTYPE HTML>
<html>

<head>
	<title>Contoh Fungsi Hitung</title>
</head>

<body>

	<?php
	$pilihan = "";
	if ($_SERVER["REQUEST_METHOD"] == "POST") {
		$pilihan = test_input($_POST["pilihan"]);
	}

	function test_input($data)
	{
		$data = trim($data);
		$data = stripslashes($data);
		$data = htmlspecialchars($data);
		return $data;
	}
	?>

	<?php
	echo "Menu<br>";
	echo "1. Pertambahan" . "<br>";
	echo "2. Pengurangan" . "<br>";
	echo "3. Perkalian	" . "<br>";
	echo "4. Pembagian	" . "<br>" . "<br>";
	?>
	<form method="post" action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>">
		Masukkan Pilihan : <input type="text" name="pilihan">
		<br>
		<br>
		<input type="submit" name="submit" value="Submit">
	</form>

	<?php
	if ($pilihan == "1") {
		header("location: pertambahan.php", true, 301);
	} elseif ($pilihan == "2") {
		header("location: pengurangan.php", true, 301);
	} elseif ($pilihan == "3") {
		header("location: perkalian.php", true, 301);
	} elseif ($pilihan == "4") {
		header("location: pembagian.php", true, 301);
	} else {
		echo " ";
	}
	?>


</body>

</html>

File: Kotlin/Contoh Fungsi/Fungsi (Functions) Hitung.kt
import java.util.Scanner

fun main(args: Array<String>) {
        var Input1 = Scanner(System.`in`)
        var Input2 = Scanner(System.`in`)

        println("Menu\n")
        println("1. Pertambahan ")
        println("2. Pengurangan ")
        println("3. Perkalian   ")
        println("4. Pembagian   \n")
        print("Masukkan Pilihan : ")
        var pilihan:Int = Input1.nextInt()

        if(pilihan == 1){
                println("\nSilahkan Masukkan Angka\n")
                print("Angka : ")
                var a:Int = Input2.nextInt()
                print("Angka : ")
                var b:Int = Input2.nextInt()
                pertambahan(a, b)
        }
        else if(pilihan == 2){
                println("\nSilahkan Masukkan Angka\n");
                print("Angka : ")
                var a:Int = Input2.nextInt()
                print("Angka : ")
                var b:Int = Input2.nextInt()
                pengurangan(a, b)
        }
        else if(pilihan == 3){
                println("\nSilahkan Masukkan Angka\n")
                print("Angka : ")
                var a:Int = Input2.nextInt()
                print("Angka : ")
                var b:Int = Input2.nextInt()
                perkalian(a, b)
        }
        else if(pilihan == 4){
                println("\nSilahkan Masukkan Angka\n")
                print("Angka : ")
                var a:Int = Input2.nextInt()
                print("Angka : ")
                var b:Int = Input2.nextInt()
                pembagian(a, b)
        }
        else{
                println("Masukkan pilihan (1..4)")
        }
}


fun pertambahan(a: Int, b:Int) {
        println("--------- +")
        println("Hasil : " + (a+b))
}

fun pengurangan(a: Int, b:Int) {
        println("--------- -")
        println("Hasil : " + (a-b))
}
fun perkalian(a: Int, b:Int) {
        println("--------- *")
        println("Hasil : " + (a*b))
}
fun pembagian(a: Int, b:Int) {
        println("--------- :")
        println("Hasil : " + (a/b))
}